File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/ADMS2015U01MenuInfo.java
package nta.med.data.model.ihis.adma;

import java.math.BigDecimal;

public class ADMS2015U01MenuInfo {
	private String sysId; 
	private String trId;
	private Double trSeq;
	private String pgmId;
	private String upprMenu;  
	private String pgmNm;
	private String pgmTp;
	private BigDecimal selectFlg;
	public ADMS2015U01MenuInfo(String sysId, String trId, Double trSeq,
			String pgmId, String upprMenu, String pgmNm, String pgmTp,
			BigDecimal selectFlg) {
		super();
		this.sysId = sysId;
		this.trId = trId;
		this.trSeq = trSeq;
		this.pgmId = pgmId;
		this.upprMenu = upprMenu;
		this.pgmNm = pgmNm;
		this.pgmTp = pgmTp;
		this.selectFlg = selectFlg;
	}
	public String getSysId() {
		return sysId;
	}
	public void setSysId(String sysId) {
		this.sysId = sysId;
	}
	public String getTrId() {
		return trId;
	}
	public void setTrId(String trId) {
		this.trId = trId;
	}
	public Double getTrSeq() {
		return trSeq;
	}
	public void setTrSeq(Double trSeq) {
		this.trSeq = trSeq;
	}
	public String getPgmId() {
		return pgmId;
	}
	public void setPgmId(String pgmId) {
		this.pgmId = pgmId;
	}
	public String getUpprMenu() {
		return upprMenu;
	}
	public void setUpprMenu(String upprMenu) {
		this.upprMenu = upprMenu;
	}
	public String getPgmNm() {
		return pgmNm;
	}
	public void setPgmNm(String pgmNm) {
		this.pgmNm = pgmNm;
	}
	public String getPgmTp() {
		return pgmTp;
	}
	public void setPgmTp(String pgmTp) {
		this.pgmTp = pgmTp;
	}
	public BigDecimal getSelectFlg() {
		return selectFlg;
	}
	public void setSelectFlg(BigDecimal selectFlg) {
		this.selectFlg = selectFlg;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/ADMS2015U01MenuInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    [Serializable]
	public class ADMS2015U01MenuInfo
	{
		private String _sysId;
		private String _trId;
		private String _trSeq;
		private String _pgmId;
		private String _upprMenu;
		private String _pgmNm;
		private String _pgmTp;
		private String _selectFlg;

		public String SysId
		{
			get { return this._sysId; }
			set { this._sysId = value; }
		}

		public String TrId
		{
			get { return this._trId; }
			set { this._trId = value; }
		}

		public String TrSeq
		{
			get { return this._trSeq; }
			set { this._trSeq = value; }
		}

		public String PgmId
		{
			get { return this._pgmId; }
			set { this._pgmId = value; }
		}

		public String UpprMenu
		{
			get { return this._upprMenu; }
			set { this._upprMenu = value; }
		}

		public String PgmNm
		{
			get { return this._pgmNm; }
			set { this._pgmNm = value; }
		}

		public String PgmTp
		{
			get { return this._pgmTp; }
			set { this._pgmTp = value; }
		}

		public String SelectFlg
		{
			get { return this._selectFlg; }
			set { this._selectFlg = value; }
		}

		public ADMS2015U01MenuInfo() { }

		public ADMS2015U01MenuInfo(String sysId, String trId, String trSeq, String pgmId, String upprMenu, String pgmNm, String pgmTp, String selectFlg)
		{
			this._sysId = sysId;
			this._trId = trId;
			this._trSeq = trSeq;
			this._pgmId = pgmId;
			this._upprMenu = upprMenu;
			this._pgmNm = pgmNm;
			this._pgmTp = pgmTp;
			this._selectFlg = selectFlg;
		}

	}
}